helm/hpcc/values.schema.json

```diff
@@ -555,6 +603,12 @@
         "spillPlane": {
           "description": "The storage plane to write spill files to",
           "type": "string"
+        },
+        "resources": {
+          "$ref": "#/definitions/resources"
+        },
+        "childContainersResources": {
```

Comment:
I think this may need to become masterContainerResources and slaveContainerResources

Reply:
worker, not slave...

Reply:
I'm happy to rename to workerContainerResources.  

However, we shouldn't be using "masterContainerResources". "resources" is the standard name for the limits for resources started with k8s.  The "resources" object is copied into the deployment automatically and nothing more needs to be done by the template coding.   If we were to rename "resources" to masterContainerResources, we'd need to rename masterContainerResources to "resources" when generating the deployment.  Furthermore,  we'd be asking users to configure the hpcc pods  in a non-standard way.